blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -188,38 +191,43 @@ const Results = ({
 	const onReadMoreClick = useCallback(() => {
 		setQueryOffset((oldOffset) => oldOffset + configuredSize);
 	}, [configuredSize, setQueryOffset]);
-
 	return viewableElements?.length > 0 && !isServerSideLazy ? (
-		<div className="results-list-container">
+		<Stack>
 			{viewableElements.map((element, index) => (
-				<ResultItem
+				<Stack
 					key={`result-card-${element._id}`}
-					ref={elementRefs[index]}
-					arcSite={arcSite}
-					element={element}
-					imageProperties={imageProperties}
-					placeholderResizedImageOptions={placeholderResizedImageOptions}
-					showByline={showByline}
-					showDate={showDate}
-					showDescription={showDescription}
-					showHeadline={showHeadline}
-					showImage={showImage}
-					showItemOverline={showItemOverline}
-					targetFallbackImage={targetFallbackImage}
-				/>
+					gap="1rem"
+					className={`${BLOCK_CLASS_NAME}__${index === 0 ? "container--first" : "container"}`}
+				>
+					<ResultItem
+						ref={elementRefs[index]}
+						arcSite={arcSite}
+						element={element}
+						imageProperties={imageProperties}
+						placeholderResizedImageOptions={placeholderResizedImageOptions}
+						showByline={showByline}
+						showDate={showDate}
+						showDescription={showDescription}
+						showHeadline={showHeadline}
+						showImage={showImage}
+						showItemOverline={showItemOverline}
+						targetFallbackImage={targetFallbackImage}
+					/>
+					<Divider />
+				</Stack>
 			))}
 			{isThereMore && (
-				<div className="see-more">
+				<Stack alignment="center" className={`${BLOCK_CLASS_NAME}__see-more`}>
```

Comment:
This `classname` is not exposed in the SCSS and is unused/not needed.

Reply:
The `Stack` is still used to center this button and I don't see an immediate need to change that, so that's fine.